examples/guides/task_specific.md

```diff
@@ -0,0 +1,22 @@
+# Task-specific setup
+
+## Bucc2018, Tatoeba
+
+The Bucc2018 and Tatoeba tasks are sentence retrieval tasks, and require the `faiss` library to run. `faiss-gpu` is recommended for speed reasons.
+
+We recommend running:
+
+```bash
+conda install faiss-gpu cudatoolkit=10.1 -c pytorch
+```
```

Comment:
What's the motivation behind not adding `faiss` to requirements.txt?

Reply:
`faiss` needs to be compiled for a specific CUDA version, separate from PyTorch. This has caused me some environment issues in the past (e.g. previously it didn't a 10.1 version, and I needed to install it via `--ignore-deps`.) I think they've added a 10.1 compatible version now (although this https://github.com/facebookresearch/faiss/issues/1295 seems to indicate to the contrary, so I'm not sure how it's working on Prince).

Given that it's only used for two tasks, and it has historically had version compatibility issues, I'm inclined to keep it off the requirements list.